### ISSUE TITLE
Faild when execute make release/image

### ISSUE THREAD

**Issue by kakascx** [cockroachdb/cockroach-operator#835]
## version
bazel 4.0.0
go 1.17.5
## reproduce
I want to build the docker image of cockroach-operator and execute `make release/image`
But failed when execute target `push_operator_image`
I modified the Makefile to obtain the debug version of the executation by following
```
.PHONY: release/image
release/image:
	# TODO this bazel clean is here because we need to pull the latest image from redhat registry every time
	# but this removes all caching and makes compile time for developers LONG.
	bazel clean --expunge
	DOCKER_REGISTRY=$(DOCKER_REGISTRY) \
	DOCKER_IMAGE_REPOSITORY=$(DOCKER_IMAGE_REPOSITORY) \
	APP_VERSION=$(APP_VERSION) \
	bazel run --stamp --verbose_failures --sandbox_debug --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
		//:push_operator_image
```
## log
```
Target //:push_operator_image failed to build
ERROR: /root/go/src/cockroach-operator/BUILD.bazel:20:15 ContainerPushDigest push_operator_image.digest failed: (Exit 1): linux-sandbox failed: error executing command 
  (cd /root/.cache/bazel/_bazel_suncx/68b161fe769f5016e338c3aae3e8d6ff/sandbox/linux-sandbox/76/execroot/com_github_coachroachdb_cockroach_operator && \
  exec env - \
    GOPROXY=https://goproxy.cn \
    PATH=/usr/local/go/bin:/root/go/bin:/home/suncx/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games \
    TMPDIR=/tmp \
  /root/.cache/bazel/_bazel_suncx/install/1a4a2fac02d50c77031d44c0d91b8920/linux-sandbox -t 15 -w /root/.cache/bazel/_bazel_suncx/68b161fe769f5016e338c3aae3e8d6ff/sandbox/linux-sandbox/76/execroot/com_github_coachroachdb_cockroach_operator -w /tmp -w /dev/shm -D -- bazel-out/k8-fastbuild/bin/cmd/cockroach-operator/ubi_update.build) linux-sandbox failed: error executing command 
  (cd /root/.cache/bazel/_bazel_suncx/68b161fe769f5016e338c3aae3e8d6ff/sandbox/linux-sandbox/76/execroot/com_github_coachroachdb_cockroach_operator && \
  exec env - \
    GOPROXY=https://goproxy.cn \
    PATH=/usr/local/go/bin:/root/go/bin:/home/suncx/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games \
    TMPDIR=/tmp \
  /root/.cache/bazel/_bazel_suncx/install/1a4a2fac02d50c77031d44c0d91b8920/linux-sandbox -t 15 -w /root/.cache/bazel/_bazel_suncx/68b161fe769f5016e338c3aae3e8d6ff/sandbox/linux-sandbox/76/execroot/com_github_coachroachdb_cockroach_operator -w /tmp -w /dev/shm -D -- bazel-out/k8-fastbuild/bin/cmd/cockroach-operator/ubi_update.build)
INFO: Elapsed time: 626.236s, Critical Path: 83.35s
INFO: 238 processes: 43 internal, 195 linux-sandbox.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
```

**Comment by kakascx** [comment]
update
I found that the container execute failed because of removing yum
![image](https://user-images.githubusercontent.com/15819504/147847836-a6dff4fd-b19a-46e5-b3d9-08b2dea27828.png)
I modified the target of container_run_and_commit_layer in /cmd/cockroach-operator/BUILD.bazel
![image](https://user-images.githubusercontent.com/15819504/147847864-9a13d342-67c8-4b1a-ba87-7da267c59bf6.png)
The problem solved by delete `microdnf remove yum`
![image](https://user-images.githubusercontent.com/15819504/147847894-26ddffde-5683-4aa9-b571-bc3e0054eb4c.png)

**Comment by pseudomuto** [comment]
Hey @kakascx, this was addressed in #842. Please feel free to re-open if you're still running into the same issue.
